cmd/configure.go

```diff
@@ -8,56 +8,113 @@ import (
 	"github.com/spf13/viper"
 )
 
-var configColors []*string
-
-var defaultColorKeys []string
-
-var defaultColors = map[string]string{
-	"gray":  "8",
-	"red":   "1",
-	"green": "2",
+// colorCfg holds config data for a single color
+// the `configValue` field is meant to be populated during the `init()` call to
+// be used by flags
+type colorCfg struct {
+	colorName    string
+	defaultValue string
+	configValue  *string
 }
 
-// configureCmd represents the configure command
+// configureCmd represents the configure command which is a container for other
+// sub-commands (e.g., colors, submission base URL)
 var configureCmd = &cobra.Command{
 	Use:   "configure",
 	Short: "Change configuration of the CLI",
+}
+
+var colorCfgs = []colorCfg{
+	{colorName: "gray", defaultValue: "8"},
+	{colorName: "red", defaultValue: "1"},
+	{colorName: "green", defaultValue: "2"},
+}
+var resetColors bool
+
+// configureColorsCmd represents the `configure colors` command for changing
+// the colors of the text output
+var configureColorsCmd = &cobra.Command{
+	Use:   "colors",
+	Short: "Change the CLI text colors",
 	Run: func(cmd *cobra.Command, args []string) {
+		if resetColors {
+			for _, cfg := range colorCfgs {
+				key := "color." + cfg.colorName
+				defaultValue := cfg.defaultValue
+				viper.Set(key, defaultValue)
+			}
+
+			viper.WriteConfig()
+			return
```

Comment:
I think we should have some user-facing message here
```suggestion
			viper.WriteConfig()
			fmt.Println("Reset colors!")
			return
```